pantheon-bundle/frontend/src/app/__snapshots__/search.test.tsx.snap

```diff
@@ -12,6 +12,7 @@ exports[`Search tests should render default Search component 1`] = `
   >
     <Pagination
       bottom={false}
+      className="search__pagination"
```

Comment:
You don't need `search__` here, but it's not a problem either.

Reply:
this is in the test file, so is it alright if I leave it